app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class='card mx-auto my-1' style='max-width: 440px;'>
+  <div class='card-body'>
+    <h5 class='card-title'> <%= link_to "Post #{i + 1}", user_post_path(@user, post) %></h5>
+    <p class='card-title'><%= post.title%></p>
+    <p class='card-text'><%= post.text%></p>
+    <span>Comments <%= post.comments_counter%></span>
+    <span>Likes <%= post.likes_counter%></span>
```

Comment:
Error in the comments counter 
  - The counter displays `15` when there are `8` comments
![image](https://github.com/PabloEGonz/blog-app/assets/78251927/ad9c5199-1a46-4bb7-94d8-6c50e7c1cb4e)

I would suggest that you use `<%= post.comments.count %>` to fix the error in the comments counter